### ISSUE TITLE
Question on _FdDataLoader

### ISSUE THREAD

**Issue by Eie1** [Rikorose/DeepFilterNet#179]
Thanks for your code .
In order to use conveniently,I change code into pytorch without dependencies.But there is a question:
Is  it possible changing _FdDataLoader into pytorch code into a class ?such as class Batch and class PytorchDataLoader.

Thanks for your answering.

**Comment by Rikorose** [comment]
Sure you can implement the data loading also in pure python.

**Comment by Eie1** [comment]
Thanks for your answering

In order to replace this: from libdfdata import _FdDataLoader in DeepFilterNet-main/pyDF-data/libdfdata/torch_dataloader.py
 all related code about  _FdDataLoader is in DeepFilterNet-main/pyDF-data/src/lib.rs ? 

Or I need to have a deep understanding of libdf ,pydf,pydf-data folders,and then change all  codes in these folders?

Whether you suggest that I use the model and loss， and my own dataloader,or change your codes of dataloader and dataset and use them?

Thanks for your helping

**Comment by Rikorose** [comment]
If you want to a close replication of the dataloader in python then you would need to essentially port the whole libDF folder do python. But you could also start with a very simple dataloader that just loads e.g. pre-mixed audios.
